fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -1208,6 +1211,36 @@ module.exports = function(config, DB) {
                 t.equal(device.callbackPublicKey, '', 'callbackPublicKey updated')
                 t.equal(device.callbackAuthKey, '', 'callbackAuthKey updated')
 
+                // Make the device a zombie, by giving it a non-existent session token
```

Comment:
The zombies *were* created when session token's got deleted without the corresponding device also being deleted. That's how the `deleteSessionToken` and `resetAccount` stored procedures used to behave.

We've got no reason to believe that any zombie sessions have been created by passing a bogus session token id to `createDevice` or `updateDevice`. Which is not to say it would definitely be wrong to add some kind of check, but that would require an extra db query to be weighed against any perceived benefit. Essentially we trust the auth server to get stuff right.

So my instinct is it's not worth it, although I'm happy to be put right if there's something i'm missing.